### PR TITLE
Addressing IANA expert review

### PR DESCRIPTION
**Pull request by emanjon**

### CODE REVIEW

File: draft-ietf-lake-edhoc.md [hunk 1]
```diff
@@ -1744,17 +1744,20 @@ The columns of the registry are Name, Label, Description, and Reference, where L
...
-+===========+=======+================+===========================+
-| kcwt      | TBD1  | COSE_Messages  | A CBOR Web Token (CWT)    |
-|           |       |                | containing a COSE_Key in  |
-|           |       |                | a 'cnf' claim             |
-+-----------+-------+----------------+---------------------------+
-| kccs      | TBD2  | map / #6(map)  | A CWT Claims Set (CCS)    |
-|           |       |                | containing a COSE_Key in  |
-|           |       |                | a 'cnf' claim             |
-+-----------+-------+----------------+---------------------------+
++------+-------+---------------+------------------------------------+
```
Comment (cabo) [inline draft-ietf-lake-edhoc.md:1747]: Looks good.

(Is there a particular reason you are not using a table here?
Just curious.)

Comment (emanjon) [reply draft-ietf-lake-edhoc.md:1747]: > Is there a particular reason you are not using a table here
No, no particular reason. None of the "tables" in the document are real tables. I thought about fixing it before when I added aasvg but did not have the time. Maybe time to fix that for -21

**Comment by gselander** [comment]
#427

**Review by gselander — APPROVED** [review]

**Comment by gselander** [comment]
Merging to post revised I-D

**Pull request merged**